server/drivers/include/irods/fileDriver.hpp

```diff
@@ -44,4 +44,5 @@ irods::error fileRegistered( rsComm_t* _comm, irods::first_class_object_ptr _obj
 irods::error fileUnregistered( rsComm_t* _comm, irods::first_class_object_ptr _object );
 irods::error fileModified( rsComm_t* _comm, irods::first_class_object_ptr _object );
 irods::error fileNotify( rsComm_t* _comm, irods::first_class_object_ptr _object , const std::string& );
+irods::error fileChecksum( rsComm_t* _comm, irods::first_class_object_ptr _object , const std::string&, std::string&);
```

Comment:
Why `fileChecksum` instead of `fileReadChecksumFromStorageDevice`?

Reply:
done